app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -5,30 +5,30 @@
 
 package org.mozilla.focus.web;
 
+import android.app.Activity;
 import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
 import android.support.annotation.NonNull;
 import android.util.AttributeSet;
 import android.view.View;
 
+import org.mozilla.focus.R;
 import org.mozilla.focus.session.Session;
 import org.mozilla.gecko.GeckoView;
-import org.mozilla.gecko.GeckoViewSettings;
+import org.mozilla.gecko.GeckoSession;
+import org.mozilla.gecko.GeckoSessionSettings;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- org\.mozilla\.gecko\.GeckoSession\.\*\.